vendor/github.com/dome9/dome9-sdk-go/services/imageassurance/policy/imageassurance_policy.go

```diff
@@ -1,35 +1,42 @@
-package admission_policy
+package imageassurance_policy
```

Comment:
here is not sure why github detect a file change/modification instead of addition